manifests/logind.pp

```diff
@@ -10,21 +10,23 @@
   service{'systemd-logind':
     ensure => running,
   }
-  $systemd::logind_settings.each |$option, $value| {
-    ini_setting{
-      $option:
-        path    => '/etc/systemd/logind.conf',
-        section => 'Login',
-        setting => $option,
-        notify  => Service['systemd-logind'],
-    }
-    if $value =~ Hash {
-      Ini_setting[$option]{
-        * => $value,
+  if $systemd::logind_settings {
```

Comment:
The new default for `logind_settings` makes this moot

Reply:
from my point of view those if statements aren't a good approach. They just add a useless layer of indentation. They aren't required if the default value is an empty hash/array instead of undef.